src/test/java/com/icegreen/greenmail/imap/commands/SearchTermBuilderTest.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * #%L
+ * Alfresco Repository
+ * %%
+ * Copyright (C) 2005 - 2016 Alfresco Software Limited
+ * %%
+ * This file is part of the Alfresco software.
+ * If the software was purchased under a paid Alfresco license, the terms of
+ * the paid license agreement will prevail.  Otherwise, the software is
+ * provided under the following open source license terms:
+ *
+ * Alfresco is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Alfresco is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ * #L%
+ */
+package com.icegreen.greenmail.imap.commands;
+
+import org.junit.Test;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Locale;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class SearchTermBuilderTest {
```

Comment:
Sorry, but you are not testing SearchTermBuilder class here.
Currently you only check if SimpleDateFormat works, which we shouldn't check as it is a 3rd party class.

Reply:
Re-wrote the Class/Method accordingly: https://github.com/Alfresco/alfresco-greenmail/blob/763e4ecab4ebc7ed21fce9bed3094cec6a816182/src/test/java/com/icegreen/greenmail/imap/commands/SearchTermBuilderTest.java#L33-L43